packages/create-discord-bot/template/TypeScript/src/events/index.ts

```diff
@@ -1,7 +1,7 @@
 import type { ClientEvents } from 'discord.js';
 
-export interface Event<T extends keyof ClientEvents = keyof ClientEvents> {
+export type Event<T extends keyof ClientEvents = keyof ClientEvents> = {
```

Comment:
Why was this changed again?

Reply:
I see it was changed in https://github.com/discordjs/discord.js/pull/9570 too, so ignore this. That said, changing it here and in that PR will probably cause merge conflicts